artemis-core-client/src/main/java/org/apache/activemq/artemis/utils/AutoCreateUtil.java

```diff
@@ -47,7 +50,7 @@ public static  void autoCreateQueue(ClientSession session, SimpleString destAddr
                      .setAddress(destAddress);
                setRequiredQueueConfigurationIfNotSet(queueConfiguration,response, RoutingType.ANYCAST, selectorString, true);
                session.createQueue(queueConfiguration);
-               ActiveMQClientLogger.LOGGER.debug("The queue " + destAddress + " was created automatically");
+               logger.debug("The queue " + destAddress + " was created automatically");
```

Comment:
Unnecessary string concatenation could be avoided here.

Reply:
Yeah there are plenty of concats left..I converted a bunch when I happened to be changing a specific line already, or just had for others in a class, but definitely we didnt try to remove them all (Clebert even likes them :P). A whole other massive PR could be done around that and e.g removing various gating still existing.

Reply:
I would not do that as part of this PR. otherwise it would never get done.

Reply:
I'll go so far as to say I will work up such an improvement PR, but I'd like to keep it separate and land this one to avoid rebasing the changes yet more as other dev continues